core/controllers/beam_jobs.py

```diff
@@ -84,11 +77,10 @@ def put(self) -> None:
         job_name = (
             self.normalized_payload.get('job_name')
             if self.normalized_payload else None)
-        job_arguments = (
-            self.normalized_payload.get('job_arguments')
-            if self.normalized_payload else None)
-
-        beam_job_run = jobs_manager.run_job_sync(job_name, job_arguments)
+        beam_job_run = (
+            jobs_manager.run_job_sync(job_name)
+            if constants.EMULATOR_MODE else
+            jobs_manager.run_job_async(job_name))
```

Comment:
Not that important, but should we move this decision to the service layer and here call just `run_job`?

Reply:
Done.